src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -48,15 +48,13 @@ public function __construct(LoaderInterface $loader = null)
      *
      *     $routes->import('blog.yml', '/blog');
      *
-     * @param mixed       $resource
-     * @param string|null $prefix
```

Comment:
can you check why we advertise null here?

Reply:
The `$prefix` parameter was added in #16477 and apparently @weaverryan added `null` to the doc block because @Tobion suggested to do so. As far as I can tell, `null` does not have a special meaning here and is cast to an empty string later.

Reply:
Worth? #32240

Reply:
@Tobion Are we sure this will land in 4.4? In that case, I would revert this file.

Reply:
Not sure. So lets keep it. Also I don't see where I asked for a nullable prefix. I commented on the `$type` not the prefix. maybe it was added to the wrong variable.

Reply:
The prefix is not nullable in https://github.com/symfony/symfony/blob/f8eed91d226bae24bd39d2b2799c21ab03d4fd9a/src/Symfony/Component/Routing/RouteCollection.php#L129. So it also should not be here.

Reply:
Yeah, that's probably what happened.